### PR TITLE
improve Makefile

### PR DESCRIPTION
**Pull request by SchSeba**
* switch to the right image registry
* allow to add golang flags

### CODE REVIEW

File: .gitignore
Comment (Eoghan1232) [inline .gitignore:1]: do  we need all of this?

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
+IMAGE_REGISTRY?=ghcr.io/k8snetworkplumbingwg/
 IMAGE_VERSION?=latest
 
-IMAGE_NAME?=$(IMAGE_REGISTRY)sriov-metrics-exporter:$(IMAGE_VERSION)
+IMAGE_NAME?=$(IMAGE_REGISTRY)sriov-network-metrics-exporter:$(IMAGE_VERSION)
 IMAGE_BUILDER?=docker
 
+# Package related
+BINARY_NAME=sriov-exporter
+PACKAGE=sriov-cni
```
Comment (Eoghan1232) [inline Makefile:9]: what's this used for?

File: .gitignore
Comment (SchSeba) [reply .gitignore:1]: I just copy this one from other projects.

I think is just a template of stuff

File: Makefile (hunk 1 above)
Comment (SchSeba) [reply Makefile]: that is a bad copy paste :P

**Review by Eoghan1232 — APPROVED** [review]
lgtm :)

File: Makefile [hunk 2]
```diff
@@ -12,14 +16,26 @@ ifdef HTTPS_PROXY
 	DOCKERARGS += --build-arg https_proxy=$(HTTPS_PROXY)
 endif
 
+# Go settings
+GO = go
+GO_BUILD_OPTS ?=CGO_ENABLED=0
+GO_LDFLAGS ?= -s -w
+GO_FLAGS ?= -buildmode=pie
```
Comment (adrianchiris) [inline Makefile:23]: do we really need this ? we dont do it in other repos ?

**Review by adrianchiris — APPROVED** [review]
lgtm, just one minor nit

File: Makefile (hunk 2 above)
Comment (SchSeba) [reply Makefile:23]: I just did a copy paste :P

but sure let me remove it

**Comment by SchSeba** [comment]
Thanks folks!

merging

**Pull request merged**